### PR TITLE
check if javascript variables not null

### PR DESCRIPTION
**Pull request by 70ray**
On the Preferences page, if using tabs other than proofreading, the font preferences do not appear and javascript exceptions were shown in the developer tools console if it is open.

Sandbox at: https://www.pgdp.org/~rp31/c.branch/prefs_exceptions

### CODE REVIEW

File: userprefs.php [hunk 1]
```diff
@@ -166,19 +166,28 @@ function do_font_sample_update(font_index, size_index, layout) {
     // font_sample when they change.
     window.addEventListener('DOMContentLoaded', () => {
         for (const orientation of ['v', 'h']) {
-            document.querySelectorAll('input[name=\"' + orientation + '_fntf\"]').forEach(radio => {
-                radio.addEventListener('change', function () {
-                    do_font_sample_update(this.value, null, orientation);
+            let radios = document.querySelectorAll('input[name=\"' + orientation + '_fntf\"]');
```
Comment (chrismiceli) [inline userprefs.php:169]: querySelectorAll shouldn't return a falsy object, just an empty list per https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll. So we shouldn't need this check, right?

File: userprefs.php [hunk 2]
```diff
@@ -166,19 +166,28 @@ function do_font_sample_update(font_index, size_index, layout) {
...
+            if(radios) {
```
Comment (chrismiceli) [inline userprefs.php:170]: *minor* other places we put a space after `if`, or `for` if we want to be consistent.

**Review by chrismiceli — APPROVED** [review]

File: userprefs.php (hunk 1 above)
Comment (70ray) [reply userprefs.php:169]: You're right. I'll remove it.

File: userprefs.php (hunk 2 above)
Comment (70ray) [reply userprefs.php]: I've changed this. It would be nice to make this a javascript file so we could lint it.

Comment (chrismiceli) [reply userprefs.php]: Yeah. I wish lunters worked better on online scripts :/

**Comment by 70ray** [comment]
Sandbox updated.

**Review by chrismiceli — APPROVED** [review]

**Review by cpeel — APPROVED** [review]
Good catch - thanks for the fix!

**Review by srjfoo — APPROVED** [review]

**Pull request merged**